client/components/UI/Form/InputArray/index.tsx

```diff
@@ -6,6 +6,8 @@ import {get} from 'lodash';
 import {Button} from '../../';
 import {Row, LineInput} from '../';
 import './style.scss';
+import {notify} from 'superdesk-core/scripts/core/notify/notify';
```

Comment:
`notify` can be imported from `superdeskApi` (in the same way `confirm` is imported)